docs/developers/guides/erc20_registration.md

```diff
@@ -66,27 +70,24 @@ writing markdown files.
 
 ### Submit the Proposal to Testnet
 
-:::tip
-**Note**: For a more detailed description of how to submit a proposal to testnet, check out the [submitting guide](../../users/governance/submitting.md).
-:::
-
-To submit a proposal to testnet through the command line with [`evmosd'](../../validators/quickstart/binary.md), use the following command:
+To [submit the proposal](../../users/governance/submitting.md) to testnet through the command line with [`evmosd`](../../validators/quickstart/binary.md), use the following command with `register-erc20`:
 
 ```bash
 evmosd tx gov submit-proposal \
+  register-erc20 \
+  <erc20-address> \
   --title=<title> \
   --description=<description> \
-  --type="Text" \
   --deposit="1000000aevmos" \
   --from=<mykey> \
-  --chain-id=<testnet_chain_id>
+  --chain-id=<testnet_chain_id> \
   --node <address>
 ```
 
 However, note that if the CLI is used to create a proposal, and `description` is set using a flag, the text will be [escaped](https://en.wikipedia.org/wiki/Escape_sequences_in_C) which may have undesired effects. If the proposal creator is using markdown or line breaks it's recommended to put the proposal text into a json file and include that file as part of the CLI proposal, as opposed to individual fields in flags. The process of creating a json file containing the proposal can be found [here](../../users/governance/submitting.md#formatting-the-json-file-for-the-governance-proposal), and the CLI command for submitting the file is below:
 
 ```bash
-evmosd tx gov submit-proposal --proposal=<path_to_json>
+evmosd tx gov submit-proposal register-erc20 --proposal=<path_to_json>
```

Comment:
nit, use consistent format for the path, you used `<path/to/metadata.json>` above